build_tools/github_actions/runner/config/github-actions-runner-deregister.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=GitHub Actions Runner Remover
+DefaultDependencies=no
+Before=halt.target shutdown.target reboot.target
+
+[Service]
+User=runner
+Group=runner
+WorkingDirectory=/home/runner/config
```

Comment:
I don't think we actually care what the working directory is. Everything uses absolute paths


```suggestion
```